File: Data Structures & Algorithms/next-greater-element-i/submission-0.py
class Solution:
    def nextGreaterElement(self, nums1: list[int], nums2: list[int]) -> list[int]:

        ans = []

        for i in nums1:
            index_of_i = None
            for j in range(len(nums2)):
                if nums2[j] == i:
                    index_of_i = j
                    break
            
            greater_element_found = False
            for k in nums2[index_of_i:]:

                if k > i:
                    ans.append(k)
                    greater_element_found = True
                    break
            if not greater_element_found:
                ans.append(-1)

        return ans

File: Data Structures & Algorithms/merge-sorted-array/submission-0.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int i = 0;
        while (m < nums1.length) {
            nums1[m] = nums2[i];
            m++;
            i++;
        }
        Arrays.sort(nums1);
    }
}
